packages/context/README.md

```diff
@@ -285,6 +287,94 @@ instanceChanges.subscribe('myObjectId', callback);
 instanceChanges.unsubscribe('myObjectId', callback);
 ```
 
+### Notification Connection
+
+-   [useNofitication](#usenotification)
+    -   [documentChanges](#documentchangessubscribeobjectidinstanceid-data-documentchange)
+    -   [instanceChanges](#instancechangessubscribeobjectid-instanceids-instancechange)
+
+#### `useNofitication()`
+
+Hook used to obtain an instanceChanges instance of `Notification` and a documentChanges instance of `Notification`.
```

Comment:
This wording seems confusing.  What is the `Notification` this is referering to?